### PR TITLE
Adding a LocalDistributor test

### PR DESCRIPTION
**Pull request by mmerrell**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: java/server/test/org/openqa/grid/distributor/local/LocalDistributorTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package org.openqa.grid.distributor.local;
```
Comment (shs96c) [inline java/server/test/org/openqa/grid/distributor/local/LocalDistributorTest.java]: This package no longer exists. I suspect you left the class in by mistake.

File: java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
+package org.openqa.selenium.grid.distributor.local;
```
Comment (shs96c) [inline java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java]: Please add the project copyright at the top. `./go copyright:update` can do that for you.

File: java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
```
Comment (shs96c) [inline java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java]: This should not be needed.

File: java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        super(new SessionId(UUID.randomUUID()), uri, capabilities);
+      }
+
+      @Override
+      public HttpResponse execute(HttpRequest req) throws UncheckedIOException {
+        return new HttpResponse();
+      }
+    }
+
+    ImmutableCapabilities caps = new ImmutableCapabilities("browserName", "cheese");
```
Comment (shs96c) [inline java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java]: Prefer using "the most general interface or class that captures intent properly" on the left hand side (which is a long way of saying `Capabilities` ;) )

File: java/server/test/org/openqa/grid/distributor/local/LocalDistributorTest.java (hunk 1 above)
Comment (mmerrell) [reply java/server/test/org/openqa/grid/distributor/local/LocalDistributorTest.java]: Yep, by mistake

File: java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java (hunk 4 above)
Comment (mmerrell) [reply java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java]: Lol, that's one of my favorite pet issues... can't believe I missed one!

File: java/server/test/org/openqa/selenium/grid/distributor/remote/RemoteDistributorTest.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package org.openqa.selenium.grid.distributor.remote;
```
Comment (shs96c) [inline java/server/test/org/openqa/selenium/grid/distributor/remote/RemoteDistributorTest.java]: This doesn't seem necessary for this PR.

**Comment by mmerrell** [comment]
Ok, @shs96c, now that I've had a few minutes--I've combed through this and only included the 3 new unit tests, no extraneous classes or package imports, and it all passes. I'm hoping this will be good enough to merge. Next up, new features in the LocalDistributor!

Thanks for being patient.

File: java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java (hunk 4 above)
Comment (mmerrell) [reply java/server/test/org/openqa/selenium/grid/distributor/local/LocalDistributorTest.java]: I believe I've fixed this in the revised PR

**Comment by mmerrell** [comment]
We can close this one out--#7442 included this

**Pull request closed without merging**